content/en/docs/chart_best_practices/templates.md

```diff
@@ -31,62 +31,73 @@ For that reason, _all defined template names should be namespaced._
 Correct:
 
 ```yaml
-{{- define "nginx.fullname" }}
-{{/* ... */}}
-{{ end -}}
+{{- define "nginx.fullname" -}}
+# ...
+{{- end }}
 ```
 
 Incorrect:
 
 ```yaml
 {{- define "fullname" -}}
-{{/* ... */}}
-{{ end -}}
+# ...
+{{- end }}
 ```
+
 It is highly recommended that new charts are created via `helm create` command
 as the template names are automatically defined as per this best practice.
 
 ## Formatting Templates
 
 Templates should be indented using _two spaces_ (never tabs).
 
-Template directives should have whitespace after the opening  braces and before
+Template directives should have whitespace after the opening braces and before
 the closing braces:
 
 Correct:
-```
-{{ .foo }}
-{{ print "foo" }}
-{{- print "bar" -}}
-```
 
-Incorrect:
-```
-{{.foo}}
-{{print "foo"}}
-{{-print "bar"-}}
+```yaml
+{{- /* Conditionally create service account */}}
+{{- if .Values.serviceAccount.create }}
+apiVersion: v1
+# ...
+{{- end }}
 ```
 
-Templates should chomp whitespace where possible:
+Incorrect:
 
 ```yaml
-foo:
-  {{- range .Values.items }}
-  {{ . }}
-  {{ end -}}
+{{-/* This comment is even a syntax error */}}
```

Comment:
It's probably best to not overload the user with more than one topic at one time. Please keep this focused on the content that is trying to be conveyed. To clarify, here we only want to point out that "Template directives should have whitespace after the opening braces and before the closing braces". Anything more than that can lead to confusion for the reader.

I would also remove the dashes from this example. Again we're only trying to introduce the bare minimum here to keep the content clear and concise. Whitespace chomp syntax is introduced further in the article.

Reply:
> It's probably best to not overload the user with more than one topic at one time.

I agree - example minimized to a single line.